Maskbook/Scene/App/FIleService/Model/FileServiceUploadingItem.swift

```diff
@@ -53,16 +55,15 @@ struct FileServiceUploadingItem: Hashable {
     let provider: String
     let state: State
     let content: Data
+    let totalBytes: Double
     let uploadedBytes: Double
     let uploadDate: Date?
     let fileType: ItemType
 
     let mime: String?
     let tx: FileServiceTranscation?
-
-    var totalBytes: Double {
-        Double(content.count)
-    }
+    
+    var contentDownloadFromTX: Data? = nil
```

Comment:
We use `FileServiceUploadingItem`  to represent the information of a uploading item, is this `contentDownloadFromTX ` really associate with uploading file?

Reply:
I will change that